core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackService.java

```diff
@@ -241,6 +251,18 @@ public Stack getById(Long id) {
         return retStack;
     }
 
+    public StackView getByIdView(Long id) {
```

Comment:
getViewById should be a better name

Reply:
This is not the best name I agree, the only reason I've chosen this as I would argue that non of the devs will know about this `view feature`, but in this case if they try to find the stack by id the idea will bring this function to their attention as well.